tests/Extractor/HttpClientTest.php

```diff
@@ -303,7 +318,8 @@ public function testTimeout()
 
         $this->assertSame('Request throw exception (with no response): {error_message}', $records[3]['message']);
         // cURL error 28 is: CURLE_OPERATION_TIMEDOUT
-        $this->assertContains('cURL error 28', $records[3]['formatted']);
+        // "cURL error 28: Connection timed out after"
+        $this->assertContains('Connection timed out after', $records[3]['formatted']);
```

Comment:
The timeout is necessary and each php-http client has diferent messages  https://github.com/php-http/httplug/issues/118 so the tests will not pass with a dfferent $adapter that guzzle and curl so my previous is obsolete

Reply:
And if we really want to add a new adapter, we'll update the assertion accordingly